combat-trainer.lic

```diff
@@ -197,17 +197,16 @@ class LootProcess
 
     @loot_specials = settings.loot_specials
     echo("  @loot_specials: #{@loot_specials}") if $debug_mode_ct
+    
     @dump_junk = settings.dump_junk
-    @dump_delay = 300 # in seconds
-    @dump_timer = Time.now - @dump_delay
+    echo("  @dump_junk: #{@dump_junk}") if $debug_mode_ct
+    @dump_timer = Time.now - 300
   end
 
   def execute(game_state)
-    if (Time.now - @dump_timer > @dump_delay) && @dump_junk
-      if DRRoom.room_objs.count >= 13
-        fput "DUMP JUNK"
-        @dump_timer = Time.now
-      end
+    if (Time.now - @dump_timer > 300) && @dump_junk && DRRoom.room_objs.count >= 13
```

Comment:
got rid of the embedded if statement and made it one large one.  Also adjusted for the lack of dump_delay